manifests/00-crd-extension-console-link.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: consolelinks.console.openshift.io
+  annotations:
+    displayName: ConsoleLinks
+    description: Extension for customizing OpenShift web console links
+spec:
+  # TODO:
+  # this relates to where the API resources live in openshift/api
+  # likely under /config/console, which might make it something like:
+  # extensions.console.config.openshift.io?
+  group: console.openshift.io
+  version: v1
+  versions:
+  - name: v1
+    served: true
+    storage: true
+  scope: Cluster
+  names:
+    plural: consolelinks
+    singular: consolelink
+    kind: ConsoleLink
+    listKind: ConsoleLinkList
+  additionalPrinterColumns:
+  - name: Text
+    type: string
+    JSONPath: .spec.text
+  - name: URL
```

Comment:
Not sure if we want to use name `URL` then reference `.spec.href`.  Can we choose `href` or `url`?  Seems like this could be confusing.

`Age` I get because when you `oc get <resource>` you get an `AGE` column that is based on the `creationTimestamp`, but I think that makes more sense as it is a computed value.  

We aren't computing anything, we are just swapping the label.